contracts/utils/RetireToucanCarbon.sol

```diff
@@ -0,0 +1,1625 @@
+        @param _bondDepository Bond depository address
+        @return bool
+     */
+    function addPool(
+        address _toucanPool,
+        address _router,
+        address _bondDepository
+    ) external onlyOwner returns (bool) {
+        bool result;
+
+        require(!listContains(toucanPools, _toucanPool), "Pool already added");
+        require(_toucanPool != address(0), "Pool cannot be zero address");
+
+        toucanPools.push(_toucanPool);
+        isToucanPool[_toucanPool] = true;
+        poolRouter[_toucanPool] = _router;
+        bondDepository[_toucanPool] = _bondDepository;
+
+        emit PoolAdded(_toucanPool, _router, _bondDepository);
+        return true;
+    }
+
+    /**
+        @notice Update the bond depository for a carbon pool
+        @param _toucanPool Pool being updated
+        @param _bondDepository New depository address
+        @return bool
+     */
+    function updateBondDepository(address _toucanPool, address _bondDepository)
+        external
+        onlyOwner
+        returns (bool)
+    {
+        require(isToucanPool[_toucanPool], "Not a Toucan Carbon Pool");
+
+        address oldBond = bondDepository[_toucanPool];
+        bondDepository[_toucanPool] = _bondDepository;
+        emit PoolBondChanged(_toucanPool, oldBond, bondDepository[_toucanPool]);
+        return true;
+    }
+
+    /**
+        @notice checks array to ensure against duplicate
+        @param _list address[]
+        @param _token address
+        @return bool
+     */
+    function listContains(address[] storage _list, address _token)
+        internal
+        view
+        returns (bool)
+    {
+        for (uint256 i = 0; i < _list.length; i++) {
+            if (_list[i] == _token) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /**
+        @notice Call the Uniswap routers for needed amounts on token being retired.
+        @param _purchaseToken Address of token being used to purchase the pool token.
+        @param _toucanPool Address of pool token being used.
+        @param _poolAmount Amount of tokens being retired.
+        @return bool
+     */
+    function getNeededBuyAmount(
+        address _purchaseToken,
+        address _toucanPool,
+        uint256 _poolAmount
+    ) public view returns (uint256, address[] memory) {
+        address[] memory path = new address[](2);
+
+        path[0] = _purchaseToken;
+        path[1] = _toucanPool;
+
+        uint256[] memory amountIn = IUniswapV2Router02(poolRouter[_toucanPool])
+            .getAmountsIn(_poolAmount, path);
+
+        // Account for .1% default AMM slippage.
+        return ((amountIn[0] * 1001) / 1000, path);
+    }
+
+    /**
+        @notice Allow withdrawal of any unbonded fees
+        @param _token Address of token to transfer
+        @return bool
+     */
+    function feeWithdraw(address _token) public onlyOwner returns (bool) {
```

Comment:
You can drop `onlyOwner` from here and update  the recipient of `safeTransfer` to `owner()`.

Reply:
Please be considerate if this question is stupid. 

So this `fees` this function is referring to is not really a _collected fees withdrawal function_. Instead you can transfer any ERC20 token to `owner` that this contract possesses?

If yes, do you think this is a good idea? Like, the KlimaDAO team become owners of tokens they could not be interested in and maybe accountable in the future.